### ISSUE TITLE
ChangeMethodName should update static imports by name

### ISSUE THREAD

**Issue by sambsnyd** [openrewrite/rewrite#9]
```
  - org.openrewrite.java.ChangeMethodName:
      method: org.mockito.Matchers anyVararg()
      name: any
```
Should turn `import static org.mockito.Matchers.anyVararg;` into `import static org.mockito.Matchers.any;`

**Comment by sambsnyd** [comment]
For easy repro, see [ChangeMethodNameTest.changeStaticImportTest()](https://github.com/openrewrite/rewrite/commit/1162273ad0bafbe2d01059649735500ef19b5266#diff-9a3bce4ec01e0ed966c87081bf908263R250)

**Comment by jkschneider** [comment]
Fixed with 1162273ad0bafbe2d01059649735500ef19b5266.
